src/stockwatch/use_cases/configuring_base.py

```diff
@@ -3,8 +3,13 @@
 from dataclasses import dataclass, fields
 from pathlib import Path
 from typing import Any, TypeVar
+import sys
+
+if sys.version_info >= (3, 11):
+    import tomllib
+else:
+    import tomli as tomllib
```

Comment:
Should we also run the tests for python 3.11? Here is a very simple example on how to do it:
https://til.simonwillison.net/github-actions/python-3-11

But we might have to change some tox configuration?

Reply:
I propose to also test 3.11 using tox (then we can also test locally). Prepared this, but installation of dependencies fails for 3.11 (problem with cachelib), so for now made the commands in tox and in github action such that only 3.10 is tested.